organization/remuneration-policy.md

```diff
@@ -14,7 +14,7 @@ The Foundation will reimburse reasonable expenses for staff, contractors and int
 
 ## Board of directors
 
-Members of the Board of directors are not paid for their work as directors. The Foundation will reimburse directors for expenses incurred as a result of their role as director of the Foundation.
+Members of the Board of directors are not paid for their work as directors. A director may separately be hired and paid by the Foundation for work carried out in a different role (for example, Chief Executive Officer). The Foundation will reimburse directors for expenses incurred as a result of their role as director of the Foundation.
```

Comment:
Shall we perhaps make this a bit more general so that we can wait for a clearer definition of these roles.

```suggestion
Members of the Board of directors are not paid for their work as directors. A director may separately be hired and paid by the Foundation for Public Code for work carried out in a different role (for example as a manager or contractor). The Foundation will reimburse directors for expenses incurred as a result of their role as director of the Foundation.
```

Reply:
Also, we're not a Foundation, so using only the word Foundation is definitely not right. We're a _vereniging_, named _Foundation for Public Code_.